packages/sonataflow-deployment-webapp/README.md

```diff
@@ -0,0 +1,168 @@
+
+```xml
+<slwtDeploymentWebapp.version>0.0.2</slwtDeploymentWebapp.version>
+```
+
+Add the webjar as a dependency in the dependencies section
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>org.webjars.npm</groupId>
+        <artifactId>sonataflow-deployment-webapp</artifactId>
+        <version>${slwtDeploymentWebapp.version}</version>
+    </dependency>
+  </dependencies>
+```
+
+Add a plugin to unpack and copy the Webjar in the plugins section
+
+```xml
+<build>
+    <plugins>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <executions>
+              <execution>
+                  <id>unpack-sonataflow-deployment-webapp</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>unpack</goal>
+                  </goals>
+                  <configuration>
+                      <artifactItems>
+                          <artifactItem>
+                              <groupId>org.webjars.npm</groupId>
+                              <artifactId>sonataflow-deployment-webapp</artifactId>
+                              <version>${slwtDeploymentWebapp.version}</version>
+                              <outputDirectory>${project.build.directory}/sonataflow-deployment-webapp</outputDirectory>
+                          </artifactItem>
+                      </artifactItems>
+                      <overWriteReleases>false</overWriteReleases>
+                      <overWriteSnapshots>true</overWriteSnapshots>
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+      <plugin>
+          <artifactId>maven-resources-plugin</artifactId>
+          <executions>
+              <execution>
+                  <id>copy-sonataflow-deployment-webapp-resources</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>copy-resources</goal>
+                  </goals>
+                  <configuration>
+                      <outputDirectory>${project.basedir}/src/main/resources/META-INF/resources</outputDirectory>
+                      <overwrite>true</overwrite>
+                      <resources>
+                          <resource>
+                              <directory
+                >${project.build.directory}/sonataflow-deployment-webapp/META-INF/resources/webjars/sonataflow-deployment-webapp/${slwtDeploymentWebapp.version}/dist</directory>
+
+                                <includes>**/*</includes>
+                          </resource>
+                      </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+```
+
+## Data.json
+
+After the installation a file `/src/main/resources/META-INF/resources/sonataflow-deploy-webapp-data.json` is needed to be created with the following text:
```

Comment:
Can you please rephrase this sentence to let people know that this is not mandatory?